CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 * [#266](https://github.com/nrepl/nrepl/pull/266): Add TLS support.
 
+### Bugs fixed
+
+* [#291](https://github.com/nrepl/nrepl/pull/291): Introduce CI support for MS-Windows. Fixes issue with completion middleware not returning values for local class files, or .jar files on windows.
```

Comment:
I think the first sentence is redundant here, as the CI support is not really a bug-fix.

Reply:
Removed.